MKOB/kobconfig.py

```diff
@@ -38,20 +38,23 @@
 
 WPM            = config.text_speed      # overall code speed
 CWPM           = config.min_char_speed  # minimum character speed (Farnsworth)
-Spacing        = 1 if config.spacing == 'WORD' else 0
+Spacing        = 1 if config.spacing == config.Spacing.word else 0
                                         # 0: character, 1: word (Farnsworth spacing)
-CodeType       = 1 if config.code_type == 'INTERNATIONAL' else 0
+CodeType       = 1 if config.code_type == config.CodeType.international else 0
                                         # 0: American, 1: International
```

Comment:
Seeing this in use, I think we should change the definition of these enums so the 'name' is the text and the 'value' is the number. That way you can just use `config.spacing.value` and `config.code_type.value` rather than having to do this test-&-conversion.

As long as we establish a standard for these, it will make it easier for all applications that use the values.

I will create an issue, make that change, and we can use that here.

Reply:
I created #97 to adjust the enums. I think this is good 'as-is' (for now) and we can adjust to make this simpler for this (and other applications) when we implement the change for #97.